### PR TITLE
Prevents document zooming if still scrolling and meta keys pressed (Issue #7022)

### PR DESCRIPTION
**Pull request by ManasJayanth**
Disables zooming when the stream of events that don't have meta/ctrl
keys active at the outset, for 100ms

[UPDATE] 1000ms

### CODE REVIEW

File: web/viewer.js [hunk 1]
```diff
@@ -1873,12 +1873,12 @@ window.addEventListener('pagechange', function pagechange(evt) {
   }
 }, true);
 
+var zoomDisabled = false, zoomDisablerId;
```
Comment (Snuffleupagus) [inline web/viewer.js]: Nit: `zoomDisablerId` seems to me like a slightly strange name, how about `zoomDisabledTimeout` instead?

File: web/viewer.js [hunk 2]
```diff
@@ -1873,12 +1873,12 @@ window.addEventListener('pagechange', function pagechange(evt) {
...
 function handleMouseWheel(evt) {
   var MOUSE_WHEEL_DELTA_FACTOR = 40;
   var ticks = (evt.type === 'DOMMouseScroll') ? -evt.detail :
               evt.wheelDelta / MOUSE_WHEEL_DELTA_FACTOR;
   var direction = (ticks < 0) ? 'zoomOut' : 'zoomIn';
-
```
Comment (Snuffleupagus) [inline web/viewer.js]: Nit: unrelated change.

**Comment by timvandermeij** [comment]
This PR needs a rebase after recently landed PRs. Furthermore, please change the commit message to be equal to the PR message, since the latter is much more descriptive.

**Comment by prometheansacrifice** [comment]
@timvandermeij Done! Ready for review

**Comment by brendandahl** [comment]
Works nicely, thanks!
